src/scripts/core/visualization-container.js

```diff
@@ -74,6 +74,12 @@
             return this.ctx;
         },
 
+        setVisibility: function (visible) {
+            var node = this.layer.node();
+
+            visible ? $(node).show() : $(node).hide();
```

Comment:
isn't this jquery, which is not a contour dependency? Or have we aliased `$` elsewhere?